google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -103,6 +103,9 @@
     <component>
       <implementation-class>com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent</implementation-class>
     </component>
+    <component>
+      <implementation-class>com.google.cloud.tools.intellij.startup.ConflictingAppEnginePluginCheck</implementation-class>
```

Comment:
can this be part of the `CloudToolsPluginInit..Component`? To me it seems like this check is part of our plugin's initialization.